docs/manual_api/index.rst

```diff
@@ -7,5 +7,6 @@ API
 
    core
    edsl
+   cpp_tile_edsl
```

Comment:
I don't think this belongs here. This should be the reference guide for the API.